app/presenters/venue_presenter.rb

```diff
@@ -0,0 +1,21 @@
+class VenuePresenter
+  def initialize(event_object_array)
+    @event_array = event_object_array
+  end
+
+  def venue_events_with_artist_info
+    final_info = {}
+    if @event_array.empty?
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.